### ISSUE TITLE
Font with offset when rendering 

### ISSUE THREAD

**Issue by davidmartos96** [simple-icons/simple-icons-font#191]
### Kind of issue 

- [ ] Improvement
- [x] Bug
- [ ] Other, namely:

### Description

Hello! I've noticed an issue with the TTF font.
When using the font in Flutter, there is a small offset when rendering the font. There is a small gap at the top and it goes a bit out of bounds at the bottom.
I'm not sure where the problem is exactly, but I did try to create a TTF font with icomoon.io manually with a few of the Simple Icons SVGs and it doesn't have that problem.
So there are some differences in how both TTF are generated. Any ideas?

I've tried with the TTF files in the Github, v7 and v9, both with the same problem.

![Screenshot_2023-11-22-11-00-10-67_082aea295e0e2b19157fadadca43d2cc.jpg](https://github.com/simple-icons/simple-icons-font/assets/22084723/2515dbca-715c-48eb-80f2-2046da689074)

**Comment by ineshbose** [comment]
I feel this may be related too: https://github.com/ineshbose/simple-icons-latex/issues/2

**Comment by mondeja** [comment]
Probably related to #170

**Comment by davidmartos96** [comment]
@mondeja Here you go: https://github.com/davidmartos96/simpleicons_offset_bug

It has a README with instructions. It includes the TTF font v10 with the simplest integration as it can be. TTF loading and only one sample circular icon declared, but you can define more manually in the `lib/icons.dart` file.

Let me know if you have any issue reproducing it.

**Comment by LitoMore** [comment]
@davidmartos96 Is this issue only on Simple Icon Font? Did you try some other icon fonts?

**Comment by davidmartos96** [comment]
@LitoMore Yes, I've only experienced it with this TTF font. Other popular TTF icon fonts render correctly on Flutter.
